RELEASE_NOTES.md

```diff
@@ -8,50 +8,47 @@
 [//]: # ( # Deprecated)
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
-⚠️ This is an alpha release. Don't use it to accept payments in your live environment.
+⚠️ This is a beta release. Don't use it to accept payments in your live environment.
```

Comment:
Let's add a point for the integration guide, we can use this url for now https://docs.adyen.com/online-payments/build-your-integration/android-5-0-0-alpha-release/ but it might change from alpha to beta later

Reply:
Just got a confirmation, it will change to `beta`